docs/api/components/schemas.yaml

```
@@ -259,35 +259,27 @@ User:
         Mandatory. unless `allow_empty_password` is `true` in the database configs.
       type: string
     admin_channels:
-      description: A list of channels to explicitly grant to the user for the default collection.
+      description: A list of channels to explicitly grant to the user for the default collection. See `collection_access` for channels in named collections.
       type: array
       items:
         type: string
     all_channels:
       description: |-
-        All the channels that the user has been granted access to for the default collection.
+        All the channels that the user has been granted access to for the default collection. See `collection_access` for channels in named collections.
```

New line for each sentence

```suggestion
        All the channels that the user has been granted access to for the default collection.
        See `collection_access` for channels in named collections.
```